### PR TITLE
Add RooFit project for GSoC 2023

### PR DESCRIPTION
**Pull request by guitargeek**
Add the RooFit project about pythonizing the RooWorkspace factory language.

FYI, @lmoneta

### CODE REVIEW

File: _gsocproposals/2023/proposal_RooFit-RooWorkspacePythonization.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+project: HSF
```
Comment (hegner) [inline _gsocproposals/2023/proposal_RooFit-RooWorkspacePythonization.md:2]: I guess you want to delete this line

File: _gsocproposals/2023/proposal_RooFit-RooWorkspacePythonization.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Mentors
+  * **[Jonas Rembser](mailto:Jonas.Rembser@cern.ch)**
+  * [Lorenzo Moneta](mailto:Lorenzo.Moneta@cern.ch)
+
+## Links
+  * [ROOT](https://root.cern/)
+  * [RooFit](https://root.cern/topical/#roofit)
+  * [RooFit tutorials](https://root.cern/doc/master/group__tutorial__roofit.html) in C++ and Python
+    Look in paricular at the tutorials that **demonstrates the current way to interact with the RooWorkspace**:
```
Comment (hegner) [inline _gsocproposals/2023/proposal_RooFit-RooWorkspacePythonization.md:49]: paricular ==> particular

File: gsoc/2023/mentors.md [hunk 3]
```diff
@@ -28,10 +28,12 @@ layout: plain
 * Kilian Lieret [kl5675@princeton.edu](mailto:kl5675@princeton.edu) Princeton
 * Jose Carlos Luna [jose.carlos.luna@cern.ch](mailto:jose.carlos.luna@cern.ch) CERN
 * Edward Moyse [edward.moyse@cern.ch](mailto:edward.moyse@cern.ch) University of Massachusetts, Amherst
+* Lorenzo Moneta [lorenzo.moneta@cern.ch](mailto:lorenzo.moneta@cern.ch) CERN
```
Comment (hegner) [inline gsoc/2023/mentors.md:31]: out of order - Moneta comes before Moyse

File: _gsocproposals/2023/proposal_RooFit-RooWorkspacePythonization.md (hunk 2 above)
Comment (hegner) [inline _gsocproposals/2023/proposal_RooFit-RooWorkspacePythonization.md:49]: demonstrates -> demonstrate

**Review by hegner — CHANGES_REQUESTED** [review]

**Comment by guitargeek** [comment]
Thanks for the comments! I have updated the PR now, implementing your corrections

**Review by hegner — APPROVED** [review]
Thanks for the fixes

**Pull request merged**